api/boilerplate/views.py

```diff
@@ -1,7 +1,15 @@
 from django.contrib.auth.models import User, Group
 from rest_framework import viewsets
 from .serializers import UserSerializer, GroupSerializer
-
```

Comment:
Please keep two lines blank before the begin of `class UserViewSet` ([PEP8](https://www.python.org/dev/peps/pep-0008/))